### PR TITLE
Fix Fandango description line in GoWatchIt IA

### LINKED ISSUE #1544: GoWatchIt: Tile doesn't look right
Investigate why this is happening:
screen shot 2015-02-17 at 1 38 39 pm

It should be consistent with this look:
screen shot 2015-02-17 at 1 39 49 pm

https://duck.co/ia/view/go_watch_it

### PR DESCRIPTION
**Pull request by marianosimone**
The Fandango description line was being replaced by a cut-off one

fandango

this should fix #1544

### CODE REVIEW

File: share/spice/go_watch_it/go_watch_it.js [hunk 1]
```diff
@@ -189,7 +189,12 @@
                     item.buy_line = "Available for Purchase";
                     item.rent_line = "";
                 }
-                
+
+                // Fandango has its own suggested line
+                if(item.provider_name === "Fandango") {
+                    item.buy_line = item.suggested_line;
```
Comment (marianosimone) [inline share/spice/go_watch_it/go_watch_it.js]: this replaces what line 175 was cutting off

Comment (marianosimone) [inline share/spice/go_watch_it/go_watch_it.js]: The suggested line works just fine... I don't know if we should hardcode it instead :8ball:

**Comment by moollaza** [comment]
LGTM :+1: 

@marianosimone great catch, thanks a ton for fixing!

**Comment by mariano-simone-zocdoc** [comment]
@moollaza glad to help

**Pull request merged**